Kosystem/Services/IPersonRepository.cs

```diff
@@ -0,0 +1,19 @@
+using Kosystem.Shared;
+
+namespace Kosystem.Services
+{
+    public interface IPersonRepository
+    {
+        PersonModel CreatePerson(NewPersonModel newPerson);
+
+        bool DeletePerson(int personId);
+
+        PersonModel? UpdatePerson(UpdatePersonModel updatePerson);
```

Comment:
Codacy found an issue: [Rename the parameter 'updatePerson' so that it does not duplicate the method name.](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=6010716)